service/javadsl/client/src/main/java/com/lightbend/lagom/javadsl/client/ConfigurationServiceLocator.java

```diff
@@ -38,30 +38,34 @@
    */
   @Deprecated
   public ConfigurationServiceLocator(Configuration configuration, CircuitBreakers circuitBreakers) {
-      this(configuration.underlying(), new CircuitBreakersPanelImpl(circuitBreakers));
+    this(configuration.underlying(), new CircuitBreakersPanelImpl(circuitBreakers));
   }
 
   @Inject
   public ConfigurationServiceLocator(Config config, CircuitBreakersPanel circuitBreakersPanel) {
-      super(circuitBreakersPanel);
-      Map<String, URI> services = new HashMap<>();
-      if (config.hasPath(LAGOM_SERVICES_KEY)) {
-          Config configServices = config.getConfig(LAGOM_SERVICES_KEY);
-          for (String key: configServices.root().keySet()) {
-              try {
-                  String value = configServices.getString(key);
-                  URI uri = new URI(value);
-                  services.put(key, uri);
-              } catch (ConfigException.WrongType e) {
-                  throw new IllegalStateException("Error loading configuration for " + getClass().getSimpleName() + ". Expected lagom.services." + key + " to be a String, but was " + configServices.getValue(key).valueType(), e);
-              } catch (URISyntaxException e) {
-                  throw new IllegalStateException("Error loading configuration for  " + getClass().getSimpleName() + ". Expected lagom.services." + key + " to be a URI, but it failed to parse", e);
-              }
-          }
+    super(circuitBreakersPanel);
+
+    Map<String, URI> services = new HashMap<>();
+
+    if (config.hasPath(LAGOM_SERVICES_KEY)) {
+      Config configServices = config.getConfig(LAGOM_SERVICES_KEY);
+      for (String key: configServices.root().keySet()) {
+        try {
+          String value = configServices.getString(key);
+          URI uri = new URI(value);
+          services.put(key, uri);
+        } catch (ConfigException.WrongType e) {
+          throw new IllegalStateException("Error loading configuration for " + getClass().getSimpleName() + ". Expected lagom.services." + key + " to be a String, but was " + configServices.getValue(key).valueType(), e);
+        } catch (URISyntaxException e) {
+          throw new IllegalStateException("Error loading configuration for  " + getClass().getSimpleName() + ". Expected lagom.services." + key + " to be a URI, but it failed to parse", e);
+        }
       }
-      this.services = HashTreePMap.from(services);
+    }
+    this.services = HashTreePMap.from(services);
   }
 
+
+
```

Comment:
Looks like this file got inadvertently reformatted. I think it's better to stick with 4 space indentation in Java files, as is the convention in most projects. I know that the Lagom codebase is inconsistent right now, but I'd rather standardize on 4-space for Java, 2-space for Scala. Most IDEs have that as default anway.